src/components/button/Button.js

```diff
@@ -1,14 +1,30 @@
-import React from 'react';
+import React from "react";
 
-import './Styles.scss';
+import "./Styles.scss";
 
 const Button = (props) => {
+  const {
+    inactive,
+    rightIconPath,
+    handleSubmit,
+    label,
+    leftIconPath,
+    loading,
+  } = props;
 
+  const leftIcon = leftIconPath;
+  const rightIcon = (
```

Comment:
We either need to rename this lo 'loaderIcon' or something like that or, better would be, to make it accept any icon and keep the name generic.